src/styles/global.css

```diff
@@ -98,31 +114,32 @@
   }
 
   .fade-out {
-	  animation: fadeOut 0.3s ease-in;
-  	opacity: 0;
+    animation: fadeOut 0.3s ease-in;
+    opacity: 0;
   }
 
   @keyframes fadeOut {
     from {
-    	opacity: 1;
+      opacity: 1;
     }
     to {
-   	  opacity: 0;
+      opacity: 0;
     }
   }
 
   .button-hover-pulse:hover {
-    animation: buttonPulse 3s cubic-bezier(0.4, 0, 0.6, 1) infinite;
+    animation: buttonPulse 1s cubic-bezier(0.4, 0, 0.6, 1) infinite;
```

Comment:
Maybe meet in the middle at 2s here, or make a separate `.button-hover-fast-pulse` class.